gsadjust/GSadjust.py

```diff
@@ -358,6 +358,21 @@ def deltas_update_not_required(self):
         self.label_deltas_update_required.setPixmap(self.update_not_needed_icon)
         self.label_deltas_update_required.setToolTip('Delta table is up to date')
 
+    def select_first_treeview_item(self):
```

Comment:
This feels like a general method "reset the view" so I've moved it to this. It's nice to keep this sort of code out of your init path, so it's easy to follow through what's happening.